### PR TITLE
Remove performance testing stuff from tools/

### PR DESCRIPTION
**Pull request by howardjohn**
This now lives in the tools repository

### CODE REVIEW

File: tools/rules.yml [hunk 1]
```diff
@@ -1,57 +0,0 @@
-subject: namespace:ns
```
Comment (kyessenov) [inline tools/rules.yml]: nit: is there a replacement for this set-up in tools/?

Comment (howardjohn) [reply tools/rules.yml]: I don't think so. What is this? There is no reference to this file in any repo and hasn't been touched in over a year

Comment (howardjohn) [reply tools/rules.yml]: Do you need this?

**Comment by howardjohn** [comment]
not stale

**Comment by howardjohn** [comment]
I still think this is relevant but going to close since it isn't getting traction

**Pull request closed without merging**